projects/ids-enterprise-ng/src/lib/busyindicator/soho-busyindicator.directive.ts

```diff
@@ -215,33 +200,15 @@ export class SohoBusyIndicatorDirective implements AfterViewInit, AfterViewCheck
       // defined by the plug-in, but in this case it is 'busyindicator'.
       this.busyindicator = this.jQueryElement.data('busyindicator');
 
-      // Initialise any event handlers.
-      this.hookupRegisteredEvents();
+      // if no events are registered then all event will be bound for backward compatibility.
```

Comment:
shouldn't need this comment either since it implicitly references the registeredEvents